src/dtgtk/culling.c

```diff
@@ -1684,7 +1685,7 @@ void dt_culling_full_redraw(dt_culling_t *table, gboolean force)
       }
       l = g_list_next(l);
     }
-    if(!in_list && table->list && g_list_length(table->list) > 0)
+    if(!in_list && table->list && table->list)
```

Comment:
no need to double check ;)

Reply:
Oops, wasn't paying close enough attention :-)